pkg/rule/postgres.go

```diff
@@ -0,0 +1,498 @@
+package rule
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+
+	"github.com/lifesum/configsum/pkg/errors"
+	"github.com/lifesum/configsum/pkg/pg"
+)
+
+const (
+	pgRuleCreateSchema = `CREATE SCHEMA IF NOT EXISTS rule`
+	pgRuleCreateTable  = `
+		CREATE TABLE IF NOT EXISTS rule.rules(
+			id TEXT NOT NULL PRIMARY KEY,
+			active BOOLEAN NOT NULL,
+			activated_at TIMESTAMP WITHOUT TIME ZONE DEFAULT (now() AT TIME ZONE 'utc'),
```

Comment:
As we deliberately want to set `activated_at` at specific point in the lifecycle of a rule it shouldn't have a default.